tests/middleware/test_wsgi.py

```diff
@@ -66,7 +68,7 @@ def test_wsgi_exception(test_client_factory):
     # The HTTP protocol implementations would catch this error and return 500.
     app = WSGIMiddleware(raise_exception)
     client = test_client_factory(app)
-    with pytest.raises(RuntimeError):
+    with pytest.raises(RuntimeError), convert_excgroups():
```

Comment:
We can use the `ExceptionGroup` here. I'm fine with this break, since we intend to remove `WSGIMiddleware`.